app/views/declaration/exporter_details.scala.html

```diff
@@ -103,11 +97,10 @@
         @govukFieldset(Fieldset(
             attributes = Map("id" -> "details"),
                 legend = Some(Legend(
```

Comment:
Fix suggested by Patryk.  This is now the same pattern as other eori & address forms e.g. carrier_details